keras_retinanet/preprocessing/pascal_voc.py

```diff
@@ -133,8 +133,8 @@ def __parse_annotation(self, element):
 
     def __parse_annotations(self, xml_root):
         size_node = _findNode(xml_root, 'size')
-        width     = _findNode(size_node, 'width',  'size.width',  parse=float)
-        height    = _findNode(size_node, 'height', 'size.height', parse=float)
+        width     = _findNode(size_node, 'width',  'size.width',  parse=float)  # noqa: F841
+        height    = _findNode(size_node, 'height', 'size.height', parse=float)  # noqa: F841
```

Comment:
Hmm, looks like we don't actually use the width and height here. Looks like we should just remove this then. @hgaiser: do you agree?

Reply:
I agree :)